### ISSUE TITLE
Any reason why 'old' branch is set to default?

### ISSUE THREAD

**Issue by antoniomtz** [sapk/docker-volume-gluster#46]
Is there any reason why the "old" branch is set as default?
I see that "Master" has support of Dep, but not in "old".

**Comment by sapk** [comment]
I try to rework all my plugins to have a common base. But this is not currently working and haven't take time for this project so I set old as the default.

**Comment by sapk** [comment]
https://github.com/sapk/docker-volume-helpers

**Comment by antoniomtz** [comment]
Yes, I see you where using Basic and Driver pkgs in Master already.
I was planning to work on [#5](https://github.com/sapk/docker-volume-gluster/issues/5) but right now all the PRs will go to 'old' branch.
What would be the best approach right now?

**Comment by sapk** [comment]
I would say it depends if it is a fix you can send it to old if a feature or want to make master work to master.
